src/app/components/mno-local-products/mno-local-product.html

```diff
@@ -70,7 +70,27 @@ <h3>{{ 'mno_enterprise.templates.dashboard.marketplace.show.price' | translate }
                 <b>{{plan.name}}</b>: {{plan.description}}
                 <br>
                 <span ng-if="!vm.isPublic && !vm.planAvailableForCurrency(plan)">{{ 'mno_enterprise.templates.dashboard.marketplace.show.no_pricing_found' | translate }}</span>
-                <span ng-if="!vm.isPublic" ng-repeat="price in plan.prices | filterBy: ['currency']: vm.orgCurrency">{{price.currency}} {{price.price_cents / 100}}</span>
+                <span ng-if="!vm.isPublic">
```

Comment:
This html from lines 73-93 can be refactored to be dried up a bit: 
```
<span ng-if="!vm.isPublic" ng-switch on="plan.pricing_type">
  <span ng-switch-when="one_off|recurring" ng-switch-when-separator="|" ng-repeat="price in plan.prices | filterBy: ['currency']: vm.orgCurrency">
    {{price.currency}} {{price.price_cents / 100}}
  </span>
  <strong ng-switch-when="recurring">&nbsp;/&nbsp;{{plan.per_unit}}&nbsp;/&nbsp;{{plan.per_duration}}</strong>
  <strong ng-switch-when="one_off">&nbsp;/&nbsp;{{'mno_enterprise.templates.dashboard.marketplace.show.price.pricing_plan.type.one_off' | translate}}</strong>
  <strong ng-switch-when="payg">{{'mno_enterprise.templates.dashboard.marketplace.show.price.pricing_plan.type.usage_based' | translate}}</strong>
</span>
```
Also note that I removed the slash from the payg message on line 89 as there will not be a price displayed before the slash.

The other thing you may notice if you are testing again is that the message on line 72 is displayed for payg plans when it should not be. This will be fixed when this PR is merged and deployed: https://github.com/maestrano/mno-enterprise-angular/pull/448. 

In the meantime, maybe just throw in a quick fix on line 72 until that is merged:
```
<span ng-if="!vm.isPublic && !vm.planAvailableForCurrency(plan) && plan.pricing_type != 'payg'">{{ 'mno_enterprise.templates.dashboard.marketplace.show.no_pricing_found' | translate }}</span>
```